src/app/views/authentication/profile/Profile.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import { IProfileProps } from '../../../../types/profile';
```

Comment:
Add a new line here.  

We separate node_module and local imports with a new line

Reply:
👍